.github/CODEOWNERS

```diff
@@ -77,6 +77,12 @@ net/proxy_resolution/ @brave/tor-reviewers
 chromium_src/**/extensions/**/*_features.json @brave/sec-team
 chromium_src/tools/json_schema_compiler/ @brave/sec-team
 
+# Browser commands available via Brave Education content
```

Comment:
these should be moved to the end of the list since they are security critical and also very specific

Reply:
Fixed in https://github.com/brave/brave-core/pull/26154/commits/b8754b2a9bd54be5fe0c2216ea7684b5bd523bad